www/base/src/styles/animations.less

```diff
@@ -38,6 +38,19 @@ tl;dr is:
   }
 }
 
+/* animation for  */
+.anim-fade {
+    -webkit-transition: .5s linear all;
```

Comment:
-webkit-transition is the only supported way by a number of non auto-upgrading mobile browsers

while -moz, -o are auto-upgrading, and are fully supporting css3 transition